cfn-resources/global-cluster-config/test/global-cluster-config.sample-template.yaml

```diff
@@ -0,0 +1,58 @@
+---
+AWSTemplateFormatVersion: '2010-09-09'
+Description: MongoDB Atlas AWS CloudFormation Quickstart for MongoDB::Atlas::GlobalClusterConfig
```

Comment:
What is the difference between this file and files in example folder? 

[mongodbatlas-cloudformation-resources](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/2b6be069bb82b266a8ca4a78135ac57ea15a6a84)/[examples](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/2b6be069bb82b266a8ca4a78135ac57ea15a6a84/examples)/global-cluster-config/

Reply:
We have updated file in example folder.please review